src/nft/components/collection/ActivityCells.tsx

```diff
@@ -57,16 +59,18 @@ const AddressLink = styled(ExternalLink)`
   }
 `
 
-const formatListingStatus = (status: OrderStatus): string => {
+const formatListingStatus = (status: QueryOrderStatus | OrderStatus): string => {
   switch (status) {
-    case OrderStatus.EXECUTED:
+    case QueryOrderStatus.EXECUTED || OrderStatus.Executed:
```

Comment:
I think you can make it just take in a string and match it against `OrderStatus` and it will evaluate correctly

Reply:
QueryOrderStatus is only needed while we still have the old query method, once the flag is merged in this whole type can be deleted